src/Parsec/Parsec.csproj

```diff
@@ -21,4 +21,8 @@
         <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
         <PackageReference Include="ServiceStack.Text" Version="6.0.2" />
     </ItemGroup>
+
+    <ItemGroup>
+      <Folder Include="Shaiya\NpcQuest\EP8\" />
+    </ItemGroup>
```

Comment:
This is unnecessary since the folder is empty

Reply:
removed